### ISSUE TITLE
Debian: testpappl: avahi_client_set_host_name assertion `client' failed

### ISSUE THREAD

**Issue by OdyX** [michaelrsweet/pappl#159]
1.0.2 has a new error when running the tests in an environment without Avahi (similar to #129 ?)

```
./testpappl -c -l testpappl.log -L debug -o testpappl.output -t all
api: testpappl: client.c:909: avahi_client_set_host_name: Assertion `client' failed.
```
See e.g.: https://salsa.debian.org/printing-team/pappl/-/jobs/1486190

**Comment by michaelrsweet** [comment]
Probably the new unit tests are tweaking this. Will fix.

**Comment by michaelrsweet** [comment]
[master a274eb7] Fix testpappl abort on Linux (Issue #159)

[v1.0.x 1984f2f] Fix testpappl abort on Linux (Issue #159)
